mod_reforged/hooks/entity/tactical/humans/mercenary.nut

```diff
@@ -219,6 +219,62 @@
 			}
 		}
 
+		if (this.getBodyItem() != null)
+		{
+			local armor = this.getBodyItem();
+			local armorAttachment = null;
+
+			if (armor.getConditionMax() < 90)
```

Comment:
For this `if/else` chain please also use the method described above to simplify it with a cutoff variable. Also move the chance roll into the top if.